### PR TITLE
Issue#5376- changes to include swagger version in generated api files

### PR DESCRIPTION
**Pull request by elizabetht**
### PR checklist

- [X] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [X] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [X] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

Followed the comments given by @wing328 in issue #5376

### CODE REVIEW

**Comment by wing328** [comment]
Looks like the existing usage is broken:
```
swagger-codegen|elizabetht-issue5376-swaggerversion ⇒ java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar version
unset
```

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java [hunk 1]
```diff
@@ -127,8 +127,8 @@ private void configureGeneratorProperties() {
         config.processOpts();
         config.preprocessSwagger(swagger);
         // TODO need to obtain version from a file instead of hardcoding it
```
Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java]: @elizabetht FYI. I'll remove this TODO line later

**Comment by wing328** [comment]
Tested with both latest master and 2.3.0. Both are showing the version correctly in the updated Ruby code.

**Comment by wing328** [comment]
@elizabetht thanks for the enhancement, which has been merged into master.

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java (hunk 1 above)
Comment (elizabetht) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java]: @wing328 I can remove this as part of the other language changes I will be making for this PR.

Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java]: @elizabetht yes, please

**Comment by wing328** [comment]
@elizabetht just a second thought about the change. Shall we only move the Version.java file under swagger codegen while keep other files (e.g. Langs.java, Meta.java, ConfigHelp.java) in swagger-codegen-cli as those files are specified to the Swagger Codegen CLI only?

**Comment by wing328** [comment]
Need to roll back based on findings in https://github.com/swagger-api/swagger-codegen/commit/702f193f6191a8f6d0b59f2d34292c616e7bcc4d#commitcomment-21911796

**Comment by wing328** [comment]
Rolled back via https://github.com/swagger-api/swagger-codegen/pull/5489

**Pull request merged**